docs/guide/Project/Specification.md

```diff
@@ -3,6 +3,62 @@ title: Specification
 sidebar_position: 2
 ---
 
-## Work in progress!
+# Meta-Analysis Specification
 
-We're still working on completing the documentation for this. Please check back later.
+Now that your studyset's studies have all the necessary information (metadata, coordinates, annotations), you can configure and run a meta-analysis! 
+
+A wizard will guide you through the following key steps of meta-analysis specification.
+
+
+
+
+### 1) **Select the Algorithm and Corrector**  
+   *Choose the algorithm for your meta-analysis and, optionally, select a statistical correction method (corrector).*
+
+   Neurosynth-Compose supports several meta-analysis algorithms, powered by NiMARE, a Python library for neuroimaging meta-analysis. 
+   For an overview to the coordinate-based meta-analysis methods supported by NiMARE, [refer to this guide](https://nimare.readthedocs.io/en/stable/cbma.html). 
+   
+   Supported coordinate-based methods include:
+
+   - **MKDADensity** (default): Multi-Kernel Density Analysis. A popular density-based meta-analysis algorithm. Creates a binary sphere around each coordinate.
+   - **ALE**: Activation Likelihood Estimation, convolves coordinates with a 3D Gaussian distribution.
+   - **MKDAChi2**: Allows you to compare your meta-analysis studies with a larger reference set of studies. It tests whether a higher proportion of studies in your meta-analysis activate a specific voxel compared to a larger population of studies that were not included. For more details, [**see the MKDAChi2 Association tutorial**](/compose-docs/tutorial/advanced/mkda_association)
+
+    For each algorithm, a default set of arguments is used, but you can easily modify them. For a complete reference on available arguments, see the [NiMARE API Docs](https://nimare.readthedocs.io/en/latest/api.html#nimare-meta-meta-analytic-algorithms).
+
+Next, select a method for [multiple comparisons correction](https://nimare.readthedocs.io/en/latest/cbma.html#multiple-comparisons-correction):
+
+
+   - **FDRCorrector** (default): False Discovery Rate correction. 
+   - **FWECorrector**: Family-Wise Error Rate correction. We strongly recommend using FWECorrector for publication-quality results with 10,000 iteration if possible. This is computationl intesive, however, hence not the default option. 
```

Comment:
10,000 **iterations** if possible. This is **computationally intensive** however, hence not the default option.